packages/core/deploy/041_deploy_long_short_pair_libraries_simple_success_token.js

```diff
@@ -0,0 +1,14 @@
+const func = async function (hre) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+
+  await deploy("SimpleSuccessTokenLongShortPairFinancialProductLibrary", {
+    from: deployer,
+    log: true,
+    skipIfAlreadyDeployed: true,
+  });
+};
+module.exports = func;
+func.tags = ["LongShortPairLibraries", "lsplib", "SimpleSuccessTokenLongShortPairLibrary"];
```

Comment:
What's the difference between the lsplib tag and the LongShortPairLibraries tag?

Reply:
Sorry, I missed this being approved. I am not sure on the reason for both the lsplib and LongShortPairLibraries tag as they are included in the same files so it appears redundant. Do you want me to remove either of them? Also, I will update the file names as it looks like a few new deploy scripts were merged.